lib/connection/result/result.js

```diff
@@ -696,8 +664,7 @@ Result.prototype.getColumn = function (columnIdentifier)
  *
  * @returns {string}
  */
-Result.prototype.getStatementId = function ()
-{
+Result.prototype.getQueryId = function () {
```

Comment:
I think better to have a new function of getQueryId() to return the same thing in case getStatementId() is used by existing customer application.

Reply:
I will apply it and commit again soon.

Reply:
Applied